do/README.md

```diff
@@ -0,0 +1,48 @@
+# Testing and Building Cmake Template using Docker and Buildkite
+
+The scripts in this directory serve several purposes.  They help provide consistency
+between your build/test results locally and the results of CI.  They also provide a quick
+and simple way to build and test your code without needing to be concerned about your
+local development environment.
+
+## WARNING
+
+Running these scripts will modify the contents of your Cmake Template directory.  
+Because the contents of this repository will be volume mounted into a Docker container and your
+code will be built and tested, any changes that you wish to preserve in your `build`
+or `logs` directory should be copied elsewhere prior to running these scripts.
```

Comment:
I don't think this will generate a 'logs' folder. I believe only catkin builds o that

Reply:
Done.